src/components/navigation/navigation.module.scss

```diff
@@ -3,33 +3,47 @@
 .navigation {
   height: $navigation-height;
```

Comment:
Something is off with the navigation height. It's defined in variables as having 120px for all screens, but that's not true in the new designs. As I see it, the heights should be as such:

| | |
|--------|--------|
| Unconnected Wallet: | XS: 56, SM: 56, MD: 120, LG: 120 |
| Connected Wallet:     | XS: 64, SM: 64, MD: 120, LG: 120 |

In the layout css module, we're computing the min-height using the navigation and footers height. Guys working on the DAO back in the days introduced that [here](https://github.com/api3dao/api3-dao-dashboard/pull/200), but I have no idea why that was needed. I suspect it had to do with the bottom image from the previous design, but I'm not super confident about it.
<img width="683" alt="image" src="https://github.com/user-attachments/assets/2a379eff-6ecd-4f47-9b7b-a56aa3ce7e1b">

So bottom line is height is wrong, so we should adjust it. I'd say the way to go is remove that calculation and leave the page to compute its own height. If we do this now, at the beginning, we'll discover any issues down the road while implementing the rest of the designs. We can also come back to it if needed.

| Not Connected | Connected |
|--------|--------|
| <img width="439" alt="image" src="https://github.com/user-attachments/assets/38af784d-f46e-4e75-a624-e8aebb70d7d1"> | <img width="439" alt="image" src="https://github.com/user-attachments/assets/1f31fc17-3ac4-4e4d-963f-aec9c1883f7a"> |

Reply:
> Guys working on the DAO back in the days introduced that https://github.com/api3dao/api3-dao-dashboard/pull/200, but I have no idea why that was needed. I suspect it had to do with the bottom image from the previous design, but I'm not super confident about it.

Good observation. Initially I went with the approach of not disrupting existing layout but I agree with you that calculated height is not needed with the new designs. I have removed it. 


I have adjusted the navigation heights as well. I was putting 120px height for small screen navigation on purpose, because it was correct for the unconnected state ("Connect wallet" button was counting to that height). But you are right that this doesn't hold for the connected state so it needed some adjustment. Hopefully it's ok this time.

Reply:
I put a `min-width` there to prevent the button to collapse like on the screenshot and to keep it true to designs for the 360px screen at the same time. 

As a general comment, given the hard deadline, we might want to focus ourselves more to the bigger things, we don't optimize the app for small screens like this anyway (Although I do appreciate your thorough review 👍)